cremalab/sarah-todo
# ADASP-29: adding state to show and hide completed tasks

## Description

DESCRIPTION_HERE

### 🔗 Reference

> _Relevant links (e.g. issue, design, etc.)_

- [ADASP-29](https://crema.atlassian.net/jira/software/projects/ADASP/boards/108?selectedIssue=ADASP-29)

### 📋 Acceptance Criteria

> _Checked off by PR **Assignees**_

- [x] text says hide when completed tasks are shown, and text says show when completed tasks are hidden
- [x] arrow points up when completed tasks are shown, arrow points down when completed tasks are hidden
- [x] toggling the arrow shows/hides the completed tasks





### 👩‍🔬 Test Instructions

> _Provided by PR **Assignees**_

1. Click on "Create New Task"
2. Enter text into input.
3. Click the "Save" button; this should save the text entered in step 2 as a new task under "Active Tasks"
4. Check the checkbox of the task; this should move it to the "Completed Tasks" area.
5. Click "Hide"; this should hide the task in "Completed Tasks" area.

## 🔎 Reviewer Checklist

> _Checked off by PR **Reviewers**_




---

> ### Roles & Responsibilities
>
> #### 👨‍💻 Assignee
>
> - Initiator of this PR (be sure to set in GitHub UI)
> - Addresses feedback and change requests
> - Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
> #### 👩‍💻 Reviewer
>
> - Invited to review PR by **Assignee** (via GitHub UI)
> - Is expected to complete a review and address followup
